src/platform/site-wide/sass/modules/_m-side-nav.scss

```diff
@@ -0,0 +1,90 @@
+.va-sidenav {
```

Comment:
I am not super familiar with the design utilities, so suggestions are very much welcome! :)

Reply:
Most of my styling comes from the specs themselves via Zeplin:
![image](https://user-images.githubusercontent.com/12773166/66655446-11902c00-ec0a-11e9-9aef-4f2400d741da.png)